### ISSUE TITLE
Option for syncing hidden files, while previously documented, is unavailable

### ISSUE THREAD

**Issue by pauljherring** [owncloud/client#10390]
### Pre-submission Checks

- [X] I [checked for similar issues](https://github.com/owncloud/client/issues?q=), but could not find any. I also checked the closed issues. I could not contribute additional information to any existing issue.
- [X] I will take the time to fill in all the required fields. I know that the bug report may be dismissed otherwise due to lack of information.

### Describe the bug

As noted in https://github.com/owncloud/docs-client-desktop/issues/314, there was previously(?), supposedly, a command line option for `owncloudcmd` to sync hidden files, only the option given for it was `-h` which conflicted with `show help`.

One supposes that this functionality is present (or has been in the past) but hidden/overridden by the use of the common option for help. 

Either 
- the option if present, should be reassigned a different short option and probably a long option
- this becomes a feature request, to add the behaviour because it's not otherwise obvious how to sync hidden files from the command line

## Expected:

```
$ owncloudcmd -h /home/user/ownCloud https://user@password@owncloud.example.com
<logs including the syncing of hidden files>
```

## Observed:
```

$ owncloudcmd -h /home/user/ownCloud https://user@password@owncloud.example.com
gui.platform: adding plugin directory "/opt/ownCloud/ownCloud/bin/../lib/x86_64-linux-gnu/plugins"
Usage: owncloudcmd [options] source_dir server_url remote_folder
owncloudcmd version 3.0.0.9215- - command line client tool

Options:
  -s, --silten                      Don't be so verbose.
  --httpproxy <http://server:port>  Specify a http proxy to use.
  --trust                           Trust the SSL certification
  --exclude <file>                  Path to an exclude list [file]
<snip>
```

### Steps to reproduce the issue

See `expected behavior` above.

### Client version number

```
$ owncloudcmd -v
gui.platform: adding plugin directory "/opt/ownCloud/ownCloud/bin/../lib/x86_64-linux-gnu/plugins"
owncloudcmd ownCloud 3.0.0.9215-
https://github.com/owncloud/client/commit/457b08ed362e4c9ae430797508791ff25171c36c
Libraries Qt 5.15.2, OpenSSL 3.0.2 15 Mar 2022
Using virtual files plugin: off
ubuntu-5.15.0-56-generic
```

### Desktop environment (Linux only)

```
$ uname -a
Linux hpdesktop 5.15.0-56-generic #62-Ubuntu SMP Tue Nov 22 19:54:14 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

### Client package version and origin (Linux only)

```
$ dpkg -l owncloud-client | cat
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                  Version       Architecture Description
+++-=====================-=============-============-=====================================================
ii  owncloud-client:amd64 3.0.0+oc-9215 amd64        The ownCloud sync client - github.com/owncloud/client
```

**Comment by pwozniak89** [comment]
Yes a command line option to allow hidden files is important for me too.

**Comment by TheOneRing** [comment]
Fixed in 3.1
